platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -3276,7 +3276,13 @@
             exportWithOrder : false
         };
 
-        $scope.setDataRetrievalError = function (value) {
+        $scope.setDataRetrievalError = function (value, responseText) {
+            if(responseText) {
+                $scope.retrievalErrorText = $scope.msg(responseText.replace('key:', '').trim());
+            } else {
+               $scope.retrievalErrorText = $scope.msg('mds.error.lookupExecError');
```

Comment:
Why do we hardcode lookup execution error in case no response text is provided. Shouldn't the error/message always come from the server?